docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -13,7 +13,7 @@ A guide to interpreting mobile app error messages, understanding their meanings,
 
 It seems that you haven't connected a wallet to your account yet. Please note that in order to receive rewards, you must connect a wallet address. Here's what you need to do:
 
-1. First, learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
+1. Learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
 2. Then, follow the instructions for [joining Arbitrum Sepolia and see your $WXM tokens](/wallet/how-to-join-arbitrum-and-see-your-tokens#add-arbitrum-sepolia-network-to-metamask).
 
 Once you've completed the above steps, you'll be all set to claim your rewards. Check back tomorrow and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
```

Comment:
"Check back tomorrow" could be something more descriptive that informs the users that once he configured his wallet he will see rewards in the next run which happens daily.

"Check back tomorrow" gives the impression that we're going to add something new to the docs tomorrow :)